cs/test/FasterLogScanTests.cs

```diff
@@ -130,6 +131,57 @@ public void ScanBasicDefaultTest([Values] TestUtils.DeviceType deviceType)
             Assert.AreEqual(entryLength, currentEntry);
         }
 
+        [Test]
+        [Category("FasterLog")]
+        [Category("Smoke")]
+        public void ScanBehindBeginAddressTest([Values] TestUtils.DeviceType deviceType)
+        {
+            // Create log and device here (not in setup) because using DeviceType Enum which can't be used in Setup
+            string filename = path + "LogScanDefault" + deviceType.ToString() + ".log";
```

Comment:
You actually can use that parameter in Setup():

            DeviceType deviceType = DeviceType.LocalMemory;
            foreach (var arg in TestContext.CurrentContext.Test.Arguments)
            {
                if (arg is TestUtils.DeviceType argDeviceType)
                    deviceType = argDeviceType;
            }
            device = CreateTestDevice(deviceType, ...);